recipes/lyra/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "lyra" %}
+{% set version = "1.6.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/bfgroup/Lyra/archive/{{ version }}.tar.gz
+  sha256: a93f247ed89eba11ca36eb24c4f8ba7be636bf24e74aaaa8e1066e0954bec7e3
+
+build:
+  number: 0
+  skip: true  # [osx]
+  skip: true  # [win]
+
+  run_exports:
```

Comment:
Lyra is headed only, this run_exports isn't needed

Reply:
I'll remove, thanks.